README.md

```diff
@@ -62,6 +62,7 @@ cd guides
 yarn install
 yarn run dev
 ```
+The default port for this project will be 8000 (in windows). Site will run at http://localhost:8000.
```

Comment:
The 'running locally' information has been moved to `CONTRIBUTING.md` - please propose a change to that file instead.